retraktarr/retraktarr.py

```diff
@@ -120,21 +109,28 @@ def main():
     )
     parser.add_argument(
         "--config",
-        type=str,
-        help="Specifies configuration file",
+        action="store",
+        nargs="?",
+        const=True,
+        default=None,
+        help="If a path is provided, retraktarr will use this config file, otherwise it outputs default config location.",
     )
     args = parser.parse_args()
 
     if args.version:
         print(f"reTraktarr v{VERSION}")
         sys.exit(0)
-    if args.config:
+
+    if args.config is not True and args.config is not None:
         config_path = args.config
     else:
         config_path = (
             f'{path.expanduser("~")}{path.sep}.config{path.sep}retraktarr.conf'
         )
-    print(config_path)
+        if args.config is True:
```

Comment:
Again would it be `True`?

Reply:
Yes, when it's provided as 'retraktarr --config' to display the current default location. As mentioned above.